src/seed_data/factories/Student.factory.ts

```diff
@@ -0,0 +1,51 @@
+import { define } from "typeorm-seeding";
+import { Student } from "../../entity/Student";
+import { chooseFrom, semesters, statuses, entryTypes, startYears, endYears } from "../utils";
+
+import * as faker from 'faker';
+
+faker.seed(123);
+
+let usedNUIDs = [];
+
+define(Student, faker => {
+    const gradYear = chooseFrom(endYears);
+    const gradSem = chooseFrom(semesters);
+    const gradDateChanged = faker.random.number(1) == 1 ? true : false;
+
+    let id = Math.floor((Math.random() + 1) * 1000000000)
```

Comment:
hmmm im wondering why we're using `Math.random` here vs `faker.random` from before

Reply:
This was originally to ensure that the number was 9 digits, like the NUIDs will be, before I realized that there was a min/max option in `faker.random` -- updating accordingly